src/util/system_exception.h

```diff
@@ -0,0 +1,46 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Elizabeth Polgreen
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * An exception that is thrown when the solver is out of time/resources
+ * and is interrupted in an unsafe state.
+ */
+
+#include "cvc5_public.h"
+
+#ifndef CVC5__SYSTEM_EXCEPTION_H
+#define CVC5__SYSTEM_EXCEPTION_H
+
+#include "base/exception.h"
+#include "cvc5_export.h"
+
+namespace cvc5 {
+
+class CVC5_EXPORT SystemException : public cvc5::Exception
```

Comment:
I don't think you need CVC5_EXPORT here. Can you try not using it?

For reference, we used to export internal exceptions; now we only officially export API exceptions. 

I'm suggesting this since this code may break later if/when cvc5 internal exceptions are further refactored.